sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -313,22 +312,22 @@ def get_queue_properties(self, **kwargs):
         except HttpResponseError as error:
             process_storage_error(error)
         response.name = self.queue_name
-        return response # type: ignore
+        return response
 
     @distributed_trace
-    def set_queue_metadata(self,
-                           metadata=None,  # type: Optional[Dict[str, Any]]
```

Comment:
Just want to call this out -- is `metadata` actually type `Dict[str, Any]`? I typically only see and use `Dict[str,str]` so want to check on this point. Will go back and align/fix the type of all `metadata` in this file after consensus is reached

Reply:
`Dict[str,str]`